### PR TITLE
Broken link fixes

### PR DESCRIPTION
**Pull request by ylboerner**

### CODE REVIEW

File: ig/markdown/Ressourcen-Observation-VitalParameter.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 Innerhalb der Deutschen Basisprofile werden Profile für den Ressourcen-Typ 'Observation' spezifiziert welche genutzt werden können um Vitalparameter und Körpermaße zu kodieren.
 
-Diese Profile sind angelehnt an die Internationlen [VitalSign Profile](https://www.hl7.org/fhir/observation-vitalsigns.html) enthalten jedoch technische Korrekturen (Verbessertes Slicing und korrigierte FHIRPath-Expressions) oder Einschränkungen auf in Deutschland gebräuchliche Maßeinheiten.
+Diese Profile sind angelehnt an die Internationlen [VitalSign Profile](http://hl7.org/fhir/R4/observation-vitalsigns.html) enthalten jedoch technische Korrekturen (Verbessertes Slicing und korrigierte FHIRPath-Expressions) oder Einschränkungen auf in Deutschland gebräuchliche Maßeinheiten.
```
Comment (alexzautke) [inline ig/markdown/Ressourcen-Observation-VitalParameter.md:6]: "https" bitte

File: ig/markdown/Terminologie-NomenklaturDerRessourcen.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ## Nomenklatur der Ressourcen
 Für die Namensgebung der Terminologie-Resourcen im Rahmen dieses Leitfadens gelten folgende Regeln
 
-- CodeSystems, ValueSets und NamingSystems, die im Rahmen des Basis-Profils erstmals *als FHIR-Resourcen* publiziert werden, erhalten eine [canonical URL](codesystem.html#4.7.3.1 "Resource Identity") aus der Domäne 
+- CodeSystems, ValueSets und NamingSystems, die im Rahmen des Basis-Profils erstmals *als FHIR-Resourcen* publiziert werden, erhalten eine canonical URL aus der Domäne
```
Comment (alexzautke) [inline ig/markdown/Terminologie-NomenklaturDerRessourcen.md:4]: Link zu https://www.hl7.org/fhir/references.html#canonical einfügen?

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: ig/markdown/Ressourcen-Observation-VitalParameter.md (hunk 1 above)
Comment (ylboerner) [reply ig/markdown/Ressourcen-Observation-VitalParameter.md]: Done, merci

**Review by patrick-werner — APPROVED** [review]

File: ig/markdown/Terminologie-NomenklaturDerRessourcen.md (hunk 2 above)
Comment (ylboerner) [reply ig/markdown/Terminologie-NomenklaturDerRessourcen.md]: Definitiv, done!

**Review by alexzautke — APPROVED** [review]

**Pull request merged**